docs/build/circuit-visualization.ipynb

```diff
@@ -604,19 +1093,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**plot_state_qsphere()** options\n",
+    "#### **plot_state_qsphere()** options\n",
```

Comment:
We are missing a demo code snippet and image for this one. @javabster whom should I ping to help?